yoti-sdk-api/src/main/java/com/yoti/api/client/DigitalIdentityClient.java

```diff
@@ -73,11 +73,21 @@ public ShareSessionQrCode createShareQrCode(String sessionId) throws DigitalIden
         return identityService.createShareQrCode(sdkId, keyPair, sessionId);
     }
 
-    public Object fetchShareQrCode(String qrCodeId) {
+    /**
+     * Retrieve the sharing session QR code
+     *
+     * @param qrCodeId
+     *              ID of the QR code to retrieve
+     * @return an {@link ShareSessionQrCode}
+     *              The content of the Share Session QR code
+     * @throws DigitalIdentityException
+     *              Aggregate exception signalling issues during the call
```

Comment:
This is not a conventional javadoc format. The descriptions should be in the same line as the var names.

Also there no need to `{@link ShareSessionQrCode}` since the method signature already specifies the return type

And 
```
Aggregate exception signalling issues during the call
```
probably would be better as:
```
Thrown if the retrieval is unsuccessful
```